content/guides/output-management.md

```diff
@@ -221,6 +222,20 @@ __webpack.config.js__
   };
 ```
 
+__src/index.html__
```

Comment:
I'm okay with this. It touches on using templates, which we removed, but I think it's a good change for this guide.

👍